install/ui/src/freeipa/dns.js

```diff
@@ -294,6 +294,11 @@ return {
         height: 300,
         sections: [
             {
+                name: 'dnszone_title',
+                label: 'Select the required zone type.',
```

Comment:
Except login, all strings in Web UI are localized. This method won't localize it.  Web UI has a text module to get a string from API, resp. from translations loaded on startup.  Search for `text.get('@i18n:`